CMakeLists.txt

```diff
@@ -218,6 +218,7 @@ endif()
 
 ## Mjolnir and associated executables
 if(ENABLE_DATA_TOOLS)
+  add_compile_definitions(DATA_TOOLS)
```

Comment:
this gives us a preprocessor define which we can use in code to disable the parts of the unit test utils that rely on mjolnir (which is disabled when you set DATA_TOOLS=Off)